core/vm/instructions.go

```diff
@@ -855,7 +858,7 @@ func opPush1(pc *uint64, interpreter *EVMInterpreter, callContext *callCtx) ([]b
 		codeLen = uint64(len(callContext.contract.Code))
 		integer = new(uint256.Int)
 	)
-	*pc += 1
+	*pc++
```

Comment:
Probably shouldn't change this line as it is unrelated to the PR.

Reply:
yeah, i have a bad habit of cleaning all the things golinter complains about, I'll have to try to resist the urge to appease it lol